src/components/CcyInfoModal/CcyInfoModal.js

```diff
@@ -24,7 +24,7 @@ const CcyInfoModal = ({
   }, [isModalVisible])
 
   useEffect(() => {
-    if (!body) {
+    if (!body || !containerRef || !containerRef.current) {
```

Comment:
why do we need `|| !containerRef || !containerRef.current` here?
```suggestion
    if (!body) {
```

Reply:
sometimes app crashes without this condition